jcapi-systems.go

```diff
@@ -221,10 +232,15 @@ func (jc JCAPI) GetSystemByHostName(hostname string, withTags bool) ([]JCSystem,
 	var returnVal []JCSystem
 
 	jcSystemRec, err := jc.Post("/search/systems", jc.hostnameFilter(hostname))
+
```

Comment:
Not my code...that's from the merge, but I'll see if I can refactor that into something more acceptable.